_posts/2016-10-16-getting-started-in-scala-on-android.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+asset-type: post
+name: getting-started-in-scala-on-android
+title: Getting started in Scala on Android
+date: 2016-10-16 00:20:00 +00:00
+author: Sergio Rodrigo Royo
+canonical:
+    name: my personal blog
+    href: http://srodrigo.me/getting-started-in-scala-on-android/
+image:
+    src: /assets/img/custom/blog/2016-10-16_getting-started-in-scala-on-android/post-image.png
```

Comment:
I created the image using GIMP and my clumsy design skills :) I was about to use Android and Scala logos but the Scala one is proprietary, so I ended up creating some "clever" text.